src/main.ts

```diff
@@ -30,6 +32,18 @@ async function run(): Promise<void> {
     const tag = await git.getTag();
     const isTagDirty = await git.isTagDirty(tag);
 
+    let yamlfile: string | unknown;
+    const argv = yargs.parse(inputs.args);
+    if (argv.config) {
+      yamlfile = argv.config;
+    } else {
+      ['.goreleaser.yaml', '.goreleaser.yml'].forEach(f => {
```

Comment:
can also be without the `.`
```suggestion
      ['.goreleaser.yaml', '.goreleaser.yml', 'goreleaser.yaml', 'goreleaser.yml'].forEach(f => {
```

Reply:
Ah ok thanks will fix that! Is there a preferred order detected by the CLI?